forloop_modules/function_handlers/auxilliary/docs.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+from collections import UserDict
+
+class Docs(UserDict):
+    
+    def __init__(self, description: Optional[str] = None, parameters_description: Optional[str] = None):
+        super().__init__()
+        self.description = description
+        self.parameters_description = parameters_description
+        
+    def add(self, name: str, title: str, description: str, typ: Optional[str] = None, example: Optional[str] = None):
+        if typ is None:
+            typ = "Any"
+            
+        if example is None:
+            example = "Will be added soon..."
```

Comment:
This could be only in "docs" export to the docs in html, in FL on hover it could be omitted

Reply:
const text like this doesnt seem right